Resources/Prototypes/Entities/Clothing/Uniforms/jumpskirts.yml

```diff
@@ -41,6 +41,10 @@
     sprite: Clothing/Uniforms/Jumpskirt/ce.rsi
   - type: Clothing
     sprite: Clothing/Uniforms/Jumpskirt/ce.rsi
+  - type: Armor
```

Comment:
didnt change the normal jumpsuit or am i insane?

Reply:
You're not insane, I neglected to save my change like a dumb. Fixed it now.